.github/workflows/linters/.yaml-lint.yml

```
@@ -0,0 +1,55 @@
+  indentation:
+    level: warning
+    spaces: consistent
+    indent-sequences: true
+    check-multi-line-strings: false
+  key-duplicates: enable
+  line-length: disable
+  new-line-at-end-of-file: disable
+  new-lines:
+    type: unix
+  trailing-spaces: disable
+  line-length:
```

yamllint

[error] duplication of key "line-length" in mapping

(key-duplicates)